### ISSUE TITLE
feast apply error when changing the registry from local file to gs:// file   

### ISSUE THREAD

**Issue by SaschaHeyer** [feast-dev/feast#1490]
## Expected Behavior 
No error =)

## Current Behavior
Getting the following error

````
TypeError: update_registry_proto() missing 1 required positional argument: 'updater'
````
## Steps to reproduce
1. Open the feature_store.yaml
2. Change `registry:` to a Google Cloud Storage path (per default it is a local path using `feast init -t gcp`)
3. Run `feast apply` 

.yaml

````
project: transactions
registry: gs://feast-feature-store-data/registry.db
provider: gcp
````
### Specifications

- Version: 0.10.1
- Platform:
- Subsystem:

**Comment by woop** [comment]
Hey @SaschaHeyer, this is fixed by https://github.com/feast-dev/feast/pull/1488

Will cut shortly.

**Comment by woop** [comment]
@SaschaHeyer Please download Feast 0.10.2. The problem should be fixed.

**Comment by SaschaHeyer** [comment]
awwww this goes smoothly. Confirmed it's fixed with 0.10.2
